spec.md

```diff
@@ -51,13 +51,14 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 
 The high-level components of the spec include:
 
-* [Image Manifest](manifest.md) - Describes the components that make up a container image.
-* [Image Index](image-index.md) - An index of annotated image manifests distributed as a complete unit.
-* [Image Layout](image-layout.md) - The filesystem layout representing the contents of an image
-* [Filesystem Layers](layer.md) - changesets that describe a container's filesystem
-* [Image Configuration](config.md) - The document determining layer ordering and configuration of the image suitable for consumption by runtime-spec
-* [Descriptors](descriptor.md) - Describes the type, metadata and content address of referenced content.
+* [Image Manifest](manifest.md) - a document describing the components that make up a container image
+* [Image Index](image-index.md) - an annotated index of one or more image manifests
```

Comment:
“one or more” is not accurate; we [explicitly allow `manifests` to be empty][1].  This allows you to do things like distribute an image-layout which only provides CAS blobs.

[1]: https://github.com/opencontainers/image-spec/blame/eb02c88ef382324a1572fa8c66f836a00d865a3f/image-index.md#L26